.gitignore

```diff
@@ -105,6 +105,7 @@ celerybeat.pid
 # Environments
 .env
 .venv
+.ffsim
```

Comment:
Please revert this change, since it seems to be special to your setup. You can add your own git ignore rules to `~/.config/git/ignore` (at least on Linux).